apps/exclude_ranges_totalpower.cpp

```diff
@@ -0,0 +1,204 @@
+// program averages few FITS images of the same sizes 
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <math.h>
+#include <unistd.h>
+
+#include <vector>
+#include <iostream>
+#include <fstream> 
+#include <sstream>
+using namespace std;
+
+string infile="total_power.txt";
+string outfile="exclude_ranges.txt";
+double gThreshold=5.00;
+bool   gDebug=false;
+
+struct cTotalPower
+{
+   long int timeindex;
+   double   total_power;
+   double   median_of_median;
+   double   median_of_rmsiqr;
+   double   median;  
+   
+   double up;
+   double down;
+};
+
+int read_file( const char* filename, vector<cTotalPower>& total_power_vec )
+{
+   total_power_vec.clear();
+   
+   ifstream ifile{filename};
+   if(ifile)
+      cout << "Open OK !!!";
```

Comment:
When using `cout` and `cerr`, remember to end the line with `<< std::endl` to add the newline and flush the buffer to `stdout` or `stderr`.